omero_marshal/encode/encoders/plate.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Glencoe Software, Inc. All rights reserved.
+#
+# This software is distributed under the terms described by the LICENCE file
+# you can find at the root of the distribution bundle.
+# If the file is missing please request a copy by contacting
+# jason@glencoesoftware.com.
+#
+
+from ... import SCHEMA_VERSION
+from .annotation import AnnotatableEncoder
+from omero.model import PlateI
+
+
+class Plate201501Encoder(AnnotatableEncoder):
+
+    TYPE = 'http://www.openmicroscopy.org/Schemas/OME/2015-01#Plate'
+
+    def encode(self, obj):
+        v = super(Plate201501Encoder, self).encode(obj)
+        self.set_if_not_none(v, 'Name', obj.name)
+        self.set_if_not_none(v, 'Description', obj.description)
+        self.set_if_not_none(v, 'ColumnNamingConvention',
+                             obj.columnNamingConvention)
+        self.set_if_not_none(v, 'RowNamingConvention',
+                             obj.rowNamingConvention)
+        self.set_if_not_none(v, 'Columns', obj.columns)
+        self.set_if_not_none(v, 'Rows', obj.rows)
+        self.set_if_not_none(v, 'DefaultSample', obj.defaultSample)
```

Comment:
As discussed in the comments, this is `FieldIndex` not a new field `DefaultSample`.